static/src/javascripts/projects/common/modules/discussion/discussion-frontend.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import config from 'lib/config';
+import fastdom from 'fastdom';
+import { integerCommas } from 'lib/formatters';
+import { loadScript } from 'lib/load-script';
+import mediator from 'lib/mediator';
+import reportError from 'lib/report-error';
+import { Loader } from './loader';
+
+const loadDiscussionFrontend = (loader: Loader, opts: Object) => {
+    const updateCommentCount = (element: HTMLElement, value: string): void => {
+        const formatted = integerCommas(value);
+
+        if (formatted) {
+            fastdom.write(() => {
+                element.textContent = formatted;
+            });
+        }
+    };
+
+    const onDiscussionFrontendLoad = (emitter): void => {
+        emitter.on('error', (feature, error) => {
+            reportError(error, { feature: `discussion-${feature}` }, false);
+        });
+
+        emitter.once('comment-count', (value: string): void => {
+            if (value === 0) {
```

Comment:
if `value` is a string type `value === 0` will never be true?

Reply:
@SiAdcock [had an idea, where this code comes from - `discussion-frontend`](https://github.com/guardian/discussion-frontend/blob/5d193d6fb3e344eb75891252fc9dba3acee4c969/src/model/discussion.js#L25-L31).